packages/odyssey-react/src/components/Table/Table.tsx

```diff
@@ -100,7 +108,7 @@ const Table = React.forwardRef<Ref, Props>((props, ref) => {
       )}
     </>
   )
-}) as OdysseyTable;
+})
```

Comment:
cc @andrewberg-okta - give the changes I've included in this file a look - thank you!